src/docker-phpunit-command.js

```diff
@@ -6,6 +6,27 @@ module.exports = class DockerPhpUnitCommand extends RemotePhpUnitCommand {
         return this.config.get("docker.paths");
     }
 
+    get remotePath() {
+        for (const [localPath, remotePath] of Object.entries(this.paths)) {
+            if (!remotePath.endsWith('/')) {
+                return remotePath + '/';
+            }
+            return remotePath;
+        }
+    }
+
+    get xmlConfigFileName() {
+        if (this.config.get("xmlConfigFileName")) {
+            return this.config.get("xmlConfigFileName");
+        } 
+
+        return 'phpunit.xml';
+    }
```

Comment:
Seems odd to include this config option and only use if for docker runs. As an end user not using docker I would assume it would work for everything.

Thinking this shouldn't be part of this PR - adding a separate config file option is a separate issue. (one that frankly I'm more open too than the remote phpunit binary)